src/main/scala/firrtl/passes/Checks.scala

```diff
@@ -219,6 +232,18 @@ object CheckHighForm extends Pass {
   }
 }
 
+object CheckHighForm extends Pass with CheckHighFormLike {
+  class IllegalChirrtlMemException(info: Info, mname: String, name: String) extends PassException(
+    s"$info: [module $mname] Memory $name has not been properly lowered from Chirrtl IR.")
```

Comment:
@albert-magyar: This is defined in both `CheckHighForm` and `CheckHighFormLike`. I think this is just oversight?

Reply:
My idea was to move it from CheckHighFormLike to CheckHighForm so that there wouldn't be a paradoxically-named `CheckChirrtl.IllegalChirrtlMemException` in the class hierarchy. Looks like I left the dead code in :(

It's unfortunate that "overriding" inner classes doesn't give me nice goof-proof messages like defs.